test/index.js

```diff
@@ -0,0 +1,150 @@
+'use strict';
```

Comment:
Testing! We need to programmatically verify that our computation pieces – at least the JavaScript parts – function as expected. This can be run with `npm test` in the project root.